src/nos/core/notifications.py

```diff
@@ -160,6 +160,16 @@ def formatted(self, **kwargs) -> "Notification":
 """,
 )
 
+IVA_UNVERIFIED_TO_USER = Notification(
+    "Contact Address Invalidation",
+    """
+One of your contact addresses has been invalidated due to too many failed verification attempts.
```

Comment:
Could in principle also be reset manually by the data steward. Maybe just "due to failed verification".

Reply:
Sounds good. See [17e0b05](https://github.com/ghga-de/notification-orchestration-service/pull/14/commits/17e0b05f6ff4e8598171a8e2fc01401c3046073c) for the change. 
Should the notification also include something like "_The complete list of your current contact addresses and their verification status may be found in your account dashboard_"?

Reply:
Yes, adding that sentence may be helpful.

Reply:
However, we currently just call it "account page", not "account dashboard".